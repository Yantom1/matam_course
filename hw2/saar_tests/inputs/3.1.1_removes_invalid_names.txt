40000000, st 0, 17, 2003, 2
30000000, 100stu, 18, 2002, 3
20000000,@ stud, 19, 2001, 2
10000000, stud| e, 20, 2000, 1
50000000, s AAA, 16, 2004, 4
70000000, s 'AAA, 16, 2004, 4
