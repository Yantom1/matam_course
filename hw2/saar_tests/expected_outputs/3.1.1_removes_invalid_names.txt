50000000, s AAA, 16, 2004, 4
